bundle.js

```diff
@@ -6,7 +6,7 @@ define(function(require) {
   , httpStatusCode: require('./http-status-code')
   , interpolate: require('./interpolate')
   , into: require('./into')
-  , location: require('./location')
+  , location: require('./location')()
```

Comment:
The location module is effectively a singleton prior to these changes. I can understand how this makes testing more difficult, however it seems to me that by changing this to be an instantiated module with local state, this also breaks the API. Anything now depending on `address.location` can no longer rely on the state of this module to be the same as what's used by `address` internally. At least, that's how I read this – it's a bit difficult to follow the changes.